internal/provider/generators/schema/main.go

```diff
@@ -292,12 +297,13 @@ func (d *Downloader) Schemas() ([]*ResourceData, *DataSources, error) {
 		}
 
 		resources = append(resources, &ResourceData{
-			CloudFormationTypeSchemaFile: cfResourceSchemaFilename,
-			GeneratedAccTestsFileName:    res + "_resource_gen_test",     // e.g. "log_group_resource_gen_test"
-			GeneratedCodeFileName:        res + "_resource_gen",          // e.g. "log_group_resource_gen"
-			GeneratedCodePackageName:     svc,                            // e.g. "logs"
-			GeneratedCodePathSuffix:      fmt.Sprintf("%s/%s", org, svc), // e.g. "aws/logs"
-			TerraformResourceType:        tfResourceTypeName,
+			CloudFormationTypeSchemaFile:      cfResourceSchemaFilename,
+			GeneratedAccTestsFileName:         res + "_resource_gen_test",     // e.g. "log_group_resource_gen_test"
+			GeneratedCodeFileName:             res + "_resource_gen",          // e.g. "log_group_resource_gen"
+			GeneratedTemplateMetadataFileName: res + "_resource_gen_metadata", // e.g. "log_group_metadata_gen"
```

Comment:
If we decide to use a `/metadata` subfolder, this naming will be redundant and can likely be shortened to `_gen`.